Added Functions billing FAQ (#6196)
cloudflare/cloudflare-docs: content/pages/platform/functions/billing.md

  * For example, you will share the included 10 million Bundled requests between Workers and Pages Functions.
* The pricing will continue to be the same as stated [here](https://developers.cloudflare.com/workers/platform/pricing).

### Free Tier
* If you are currently a free Workers user, you will share the same 100,000 free tier limit pool with your Pages Functions. For example, you could use 50,000 Functions requests and 50,000 Workers requests to use your full 100,000 daily request usage.
* Today, we recommend that anyone currently using over (or near) 100,000 requests on Pages Functions, Workers or both combined should upgrade to Workers Paid.

### Enterprise
* Your Workers contract allocation will now be shared between Workers and Functions.
* If you currently have a Workers contract and need to make an addition to your contract, you can reach out to your Customer Service Manager (CSM) to make this change.
* If you do not have a Workers contract, you can reach out to your Customer Service Manager (CSM) to make this change.

> ## Have questions?
> Let us know! For any questions unanswered on this page, reach out to us on [Discord](https://discord.com/invite/cloudflaredev) in the [#functions channel](https://discord.com/channels/595317990191398933/910978223968518144) in the Pages section or contact your Customer Success Manager if applicable.

Reviewer: ```suggestion
## Have questions?

For any questions unanswered on this page, reach out to us on [Discord](https://discord.com/invite/cloudflaredev) in the [#functions channel](https://discord.com/channels/595317990191398933/910978223968518144) in the Pages section or contact your Cloudflare account team.
```

## FAQ

> ### Will requests to assets be charged?
> During the beta asset hits would also count as a Function invocation. At the start of September we implemented support for a `_routes.json` file which allows you to exclude routes from running Functions. This allows for you to exclude your static files and not have Function invocations charged. This file is generated automatically if not specified so don't worry about having to create it yourself. You should already see an accurate Function count today which does not include static asset hits. If you're still seeing a value higher than expected please [join the Discord](https://discord.gg/cloudflaredev) and let us know.

Reviewer: ```suggestion
### Will requests to assets be charged?

During the beta, asset hits would also count as a Function invocation. At the start of September we implemented support for a `_routes.json` file which allows you to exclude routes from running Functions. This allows for you to exclude your static files and not have Function invocations charged. This file is generated automatically, unless specified, so do not worry about having to create it yourself. You should already see an accurate Function count today which does not include static asset hits. If you are still seeing a value higher than expected please [join the Discord](https://discord.gg/cloudflaredev) and let us know.
```
